x-pack/plugins/console_extensions/spec/generated/security.delete_privileges.json

```diff
@@ -13,6 +13,6 @@
     "patterns": [
       "_security/privilege/{application}/{name}"
     ],
-    "documentation": "TODO"
+    "documentation": "https://www.elastic.co/guide/en/elasticsearch/reference/master/security-api-delete-privilege.html"
```

Comment:
We probably want these doc links to reference `current` instead of `master`.

Reply:
there's code that handles the linking to the right version based on the version of Kibana.